resources/lang/en/admin/hardware/form.php

```diff
@@ -11,6 +11,7 @@
     'checkout_date'		=> 'Checkout Date',
     'checkin_date'		=> 'Checkin Date',
     'checkout_to'		=> 'Checkout to',
+	'company_name'      => 'Company Name',
```

Comment:
No need to create a new string here (that all those poor bastards have to translate). We have "Company Name" in `admin/companies/table.name`